init/bitlbee.service.in

```diff
@@ -4,6 +4,9 @@ Description=BitlBee IRC/IM gateway
 [Service]
 ExecStart=@sbindir@bitlbee -F -n
 KillMode=process
+StateDirectory=bitlbee
+RuntimeDirectory=bitlbee
+User=bitlbee
```

Comment:
Debian policy requires that this username starts with an underscore, so it would be good to do here too (since bitlbee ships with a debian package).

Reply:
Hm? This sounds quite strange besides that would not a policy on other distributions.
Can you link that policy?

Reply:
https://www.debian.org/doc/debian-policy/ch-opersys.html#introduction

Reply:
Answered down below.